Services/CompanyService/appsettings.json

```diff
@@ -10,8 +10,11 @@
   "ConnectionStrings": {
     "SQLConnectionString": "Server=localhost;Database=CompanyDB;Trusted_Connection=True;"
   },
-  "ServiceInfo": {
-    "ID": "47EC83DB-F697-4212-AA50-8D10EF1F17BC",
-    "Name": "CompanyService"
+  "RabbitMQ": {
+    "Host": "localhost",
+    "Username": "CompanyService_47EC83DB-F697-4212-AA50-8D10EF1F17BC",
+    "Password": "CompanyService",
+    "AccessValidatorCheckRightsServiceURL": "rabbitmq://localhost/CheckRightsService",
+    "AccessValidatorUserServiceURL": "rabbitmq://localhost/UserService"
```

Comment:
Most likely this is true, but can you explain why this is so?

Reply:
What do you mean exactly? 
I'm negotiating with Spartak what our appsettings should look like. I'll commit the approved version. This can be ignored

Reply:
But i think this structure should be used everywhere. It provides a clearer configuration and you need to make changes only in the appsettings.json file.
Also all services that will be calling AccessValidator have to have this section.